test/integration.test.ts

```diff
@@ -591,8 +595,17 @@ describe("integration tests", () => {
         `result of running showSubnet aqua function is expected to be an array of WorkerServices, but it is: ${showSubnetResult}`,
       );
 
-      // check logs are working
-      assert((await fluence({ args: ["deal", "logs"], cwd })) !== "");
+      const logs = await fluence({ args: ["deal", "logs"], cwd });
+
+      if (logs.includes(LOGS_RESOLVE_SUBNET_ERROR_START)) {
+        throw new Error(
+          `Failed to resolve subnet when getting deal logs:\n\n${logs}`,
+        );
+      }
+
+      if (logs.includes(LOGS_GET_ERROR_START)) {
```

Comment:
It will be good to add assert that logs aren't empty also

Reply:
done